### PR TITLE
Add Chinese Translation (2023.09.18)

### PR DESCRIPTION
**Pull request by SleepyAsh0191**

### CODE REVIEW

File: dataset/dictionary/weapons.json5 [hunk 1]
```diff
@@ -376,7 +383,7 @@
   {
     en: "Bloodtainted Greatsword",
     ja: "龍血を浴びた剣",
-    zhCN: "沐浴龙血之剑",
+    zhCN: "沐浴龙血的剑",
```
Comment (SleepyAsh0191) [inline dataset/dictionary/weapons.json5:386]: https://wiki.biligame.com/ys/%E6%B2%90%E6%B5%B4%E9%BE%99%E8%A1%80%E7%9A%84%E5%89%91

**Comment by SleepyAsh0191** [comment]
@xicri @Bill-Haku

File: dataset/dictionary/weapons.json5 (hunk 1 above)
Comment (xicri) [reply dataset/dictionary/weapons.json5:386]: Confirmed, thanks!
沐浴龙血的剑

**Comment by xicri** [comment]
@SleepyAsh0191 Thanks, can you rebase by main to eliminate unnecessary merge commits?
I think `git fetch upstream && git rebase upstream/main` should work.

Let me know if you are not sure how to rebase. I will do it.

**Review by xicri — DISMISSED** [review]
Thanks, looks good!

File: dataset/dictionary/dialogue.json5 [hunk 2]
```diff
@@ -314,6 +317,7 @@
   {
     en: "Have a safe trip",
     ja: "行ってらっしゃい",
+    // TODO add zhCN
```
Comment (xicri) [inline dataset/dictionary/dialogue.json5:320]: This Wriothesley's dialogue can be found in the special program. I guess the same video was broadcasted on Bilibili.
 https://youtu.be/NUBIYZ_KQFc?si=zT6-j1PeGFNjnEVW&t=681

**Review by xicri — APPROVED** [review]
Thanks, your additional updates looks good too!
After @Bill-Haku approved, I will merge this.

File: dataset/dictionary/dialogue.json5 (hunk 2 above)
Comment (Bill-Haku) [reply dataset/dictionary/dialogue.json5:320]: I checked it out on Bilibili.

```
zhCN: "一路顺风",
```

Comment (xicri) [reply dataset/dictionary/dialogue.json5:320]: @Bill-Haku Thanks!

@SleepyAsh0191 Can you add this?

**Review by Bill-Haku — APPROVED** [review]
@xicri, You can merge it after @SleepyAsh0191 appended that.

File: dataset/dictionary/dialogue.json5 (hunk 2 above)
Comment (SleepyAsh0191) [reply dataset/dictionary/dialogue.json5]: done

**Review by xicri — APPROVED** [review]

File: dataset/dictionary/dialogue.json5 (hunk 2 above)
Comment (xicri) [reply dataset/dictionary/dialogue.json5]: @SleepyAsh0191 Thanks, merged!

**Pull request merged**